styles/_header.scss

```diff
@@ -0,0 +1,78 @@
+
+/* HEADER */
+
+h1 {
+  a {
+    display: block;
+    letter-spacing: 1em;
+    padding-top: 70px;
+    padding-left: 1em;
+    padding-bottom: 0.5em;
+    text-indent: -9999px;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Negative text-indent doesn't work well with RTL. If you use text-indent for image replacement explicitly set direction for that item to ltr.](https://www.codacy.com/app/j-hasseriis/cepheus/file/1863156548/issues/source?bid=3085146&fileBranchId=3086993#l11)